compiler/python_archive_test.py

```diff
@@ -135,6 +135,10 @@ def test_extract_dir(self):
         self.assertTrue(os.path.exists(self._extracted_main))
 
     def test_create_extract_dir_fallback_to_tmp(self):
+        # Root can do anything, so this test doesn't work
+        if os.geteuid() ==0:
```

Comment:
nit: missing space before the zero.

Also I assume you didn't use the [decorator approach](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) because these tests are run with python2?

Reply:
Huh, didn't know that existed for `unittest`, but it's true that these tests run with `srcs_version = "PY2AND3"` so I think I wouldn't be able to use it anyway.